index.js

```diff
@@ -806,13 +776,16 @@
   //. In general terms, `compose` performs right-to-left composition of two
   //. unary functions.
```

Comment:
We could simplify this description. The first paragraph is overly wordy. What do you think of the following?

_Composes two unary functions, from right to left. Equivalent to Haskell's `(.)` function._

That says it all, really.